README.md

```diff
@@ -36,6 +36,7 @@ Or copy it into one of these folders for installing it system-wide:
 
 * `/usr/local/lib/docker/cli-plugins` OR `/usr/local/libexec/docker/cli-plugins`
 * `/usr/lib/docker/cli-plugins` OR `/usr/libexec/docker/cli-plugins`
+* Rename the binary to `docker-compose`
```

Comment:
I think the overall phrasing starting from `Copy the relevant binary…` could be improved.
Maybe
```
Copy the relevant binary for your OS under `~/.docker/cli-plugins/docker-compose` or for installing it system-wide under one of:
- `/usr/local/lib/docker/cli-plugins/docker-compose` or `/usr/local/libexec/docker/cli-plugins/docker-compose`
- `/usr/lib/docker/cli-plugins/docker-compose` or `/usr/libexec/docker/cli-plugins/docker-compose`
```
What do you think?